pytorch_lightning/loggers/base.py

```diff
@@ -95,9 +98,9 @@ def update_agg_funcs(
     def experiment(self) -> Any:
         """Return the experiment object associated with this logger."""
 
-    def _aggregate_metrics(
-            self, metrics: Dict[str, float], step: Optional[int] = None
-    ) -> Tuple[int, Optional[Dict[str, float]]]:
+    def _aggregate_metrics(self,
+                           metrics: Dict[str, float],
+                           step: Optional[int] = None) -> Tuple[int, Optional[Dict[str, float]]]:
```

Comment:
I wonder why isn't it formatted like this:
```python
def fun(
    arg1,
    arg2,
)
```
just like others.

Reply:
well in this case I would rather expect as they are short
```
def fun(
    arg1, arg2,
) -> ... :
```
but fixed with , at the end